src/SprykerEco/Zed/NewRelic/Communication/Console/RecordDeploymentConsole.php

```diff
@@ -53,13 +53,13 @@ protected function configure(): void
 
         $this->addArgument(
             self::ARGUMENT_USER,
-            InputArgument::OPTIONAL,
+            InputArgument::REQUIRED,
```

Comment:
Isn't it a breaking change?

Reply:
As explained for the third argument. By having these arguments positional, I have to force the second one to be required as well

Reply:
It makes it a major IMO.

Reply:
ok